### PR TITLE
Make Changelog examples in CONTRIBUTING.md more obvious

### LINKED ISSUE #1049: Make the Changelog examples in CONTRIBUTING.md more obvious
At first glance (i.e. when skimming) the Changelog examples in CONTRIBUTING.md look like an actual changelog, rather than examples. This was momentarily disorienting for me. A simple formatting change should help make this more obvious.

### PR DESCRIPTION
**Pull request by calebhailey**
Signed-off-by: Caleb Hailey <calebhailey@gmail.com>

## What is this change?

Makes the Changelog examples in CONTRIBUTING.md more obvious & closes #1049. 

## Why is this change necessary?

To make the Changelog examples in CONTRIBUTING.md stand out. 

## Does your change need a Changelog entry?

Sure!

## Do you need clarification on anything?

Pretty straightforward, I hope. :) 

## Were there any complications while making this change?

No. The instructions in CONTRIBUTING.md were very helpful!

### CODE REVIEW

**Comment by calebhailey** [comment]
Did I follow the contributing guidelines correctly? 🙏 😄

**Comment by amdprophet** [comment]
This just needs a rebase and then it can be merged.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -92,31 +92,39 @@ change goes in the Changelog.
 
 Any _new_ functionality goes here. This may be a new field on a data type or a new data
 type altogether; a new API endpoint; or possibly a whole new feature. In general, these
-are sentence that start with the word "added." Examples:
+are sentence that start with the word "added."
```
Comment (grepory) [inline CONTRIBUTING.md]: sentences

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -92,31 +92,39 @@ change goes in the Changelog.
...
 - `begin` field to silences that initiates silencing at a given timestamp
 - /healthz endpoint that reports health of the sensu-agent process
 
 ### Changed
 
 Changes to any existing component or functionality of the system that does not cause
-breaking changes to users or developers go here. _Changed_ is distinguishable from 
-_Fixed_ in that it is a change to intentional functionality. Examples:
+breaking changes to users or developers go here. _Changed_ is distinguishable from
+_Fixed_ in that it is a change to intentional functionality.
```
Comment (grepory) [inline CONTRIBUTING.md]: existing instead of intentional? intentional change to existing functionality? just seems not quite right.

**Comment by grepory** [comment]
Closed in favor of #1169

**Pull request closed without merging**